src/qubx/trackers/sizers.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from qubx import logger
+from qubx.core.basics import Position, Signal
+from qubx.core.strategy import IPositionSizer, StrategyContext
+
+
+class FixedSizer(IPositionSizer):
+    """
+    Simplest fixed sizer class. It uses same fixed size for all signals.
+    We use it for quick backtesting of generated signals in most cases.
+    """
+
+    def __init__(self, fixed_size):
+        self.fixed_size = abs(fixed_size)
+
+    def get_position_size(self, ctx: StrategyContext, signal: Signal) -> float:
+        return signal.signal * self.fixed_size
+
+
+class FixedRiskSizer(IPositionSizer):
+    def __init__(self, max_cap_in_risk: float, max_allowed_position=np.inf):
+        """
+        Create fixed risk sizer calculator instance.
+        :param max_cap_in_risk: maximal risked capital (in percentage)
+        :param max_allowed_position: limitation for max position size
+        """
+        self.max_cap_in_risk = max_cap_in_risk
+        self.max_allowed_position = max_allowed_position
+
+    def get_position_size(self, ctx: StrategyContext, signal: Signal) -> float:
+        if signal.signal != 0:
+            if signal.stop and signal.stop > 0:
+                _pos = ctx.positions[signal.instrument.symbol]
+                _q = ctx.quote(signal.instrument.symbol)
+
+                _direction = np.sign(signal.signal)
+                _cap = ctx.get_capital() + max(_pos.total_pnl(), 0)
```

Comment:
Maybe it makes sense to use by default just the total capital? I added for this purpose the `get_total_capital` method in the `AccountProcessor`. So you could access it via `ctx.acc.get_total_capital()` or add another method for this in the context interface. What is your opinion on this?

Reply:
Yep makes sense. It came from old Qube code, so we can just to redo it in more accurate way.